### DISCUSSION TITLE
Problem by creation new translation

### DISCUSSION THREAD

**Discussion by rynskiy** [directus/directus#15529] [Archive]
Hi,

since last week, it is no longer possible to enter the new translations... I have tried to delete a collection and create new - same bug (Microsoft Edge Version 104.0.1293.70, Chrome Version 104.0.5112.102). Please fix these bug asap! (See screen-video).

https://user-images.githubusercontent.com/54798086/189613120-409a7ef7-015a-455e-a2bd-7f18638c47d3.mp4

Best regards,
Tony

**Comment by azrikahar** [comment]
This is a known bug currently tracked in #15460 👍

**Reply by rynskiy** [reply]
Hi @azrikahar. Is it known when it will be fixed?

**Reply by br41nslug** [reply]
[Unfortunately, we can not provide estimates on features or fixes](https://docs.directus.io/getting-started/support.html#can-you-give-an-eta-for-this-feature-fix). As of now, the only way to get an accurate timeline or release date is to [sponsor](https://docs.directus.io/getting-started/support.html#sponsored-work) this task. We recognize that this may be an important feature/fix, but we are a small open-source organization with a lot to triage and complete.

**Reply by azrikahar** [reply]
> Is it known when it will be fixed?

Just as an update, the issue has since been resolved by #15500 and it will be included in the next release (which is _after_ `9.17.4` for future readers)
